hugo/content/configuration-overview.md

```diff
@@ -0,0 +1,242 @@
+---
+title: "Configuration Overview"
+draft: false
+description: "Learn the overview of configuration"
+weight: 500
+toc: true
+tags: [ "docs" ]
+docs: "DOCS-1229"
+categories: ["configuration"]
+doctypes: ["task"]
+---
+
+## Overview
+
+The following sections explain how to configure the NGINX Agent using configuration files, CLI flags, and environment variables.
+
+{{<note>}}
+
+- The NGINX Agent interprets configuration values set by configuration files, CLI flags, and environment variables in the following priorities:
+
+  1. CLI flags overwrite configuration files and environment variable values.
+  2. Environment variables overwrite configuration file values.
+  3. Config files are the lowest priority and config settings are superseded if either of the other options is used.
+
+- You must open any required firewall ports or add SELinux/AppArmor rules for the ports and IPs you want to use.
+
+{{</note>}}
+
+### Configure with Config Files
+
+The configuration files for the NGINX Agent are `/etc/nginx-agent/nginx-agent.conf` and `/etc/nginx-agent/agent-dynamic.conf`. These files have comments at the top indicating their purpose.
```

Comment:
`/etc/nginx-agent/agent-dynamic.conf` should be `/var/lib/nginx-agent/agent-dynamic.conf`